src/pages/project/project.less

```diff
@@ -74,6 +74,13 @@
 
     &.source {
       opacity: 0.75;
+      .shim .indicator {
+        width: 50%;
+        height: 160/220 * 50%;
```

Comment:
They are the same as the ones that appear on `.target .shim .indicator`. They create square bounding box based off 160:220 aspect ratio.